snuba/migrations/migration.py

```diff
@@ -197,37 +164,75 @@ def backwards(self, context: Context, dry_run: bool) -> None:
 
     def __dry_run(
         self,
-        local_operations: Sequence[SqlOperation],
-        dist_operations: Sequence[SqlOperation],
-        local_first: bool,
+        ops: Sequence[SqlOperation],
     ) -> None:
-        def print_local() -> None:
-            print("Local operations:")
-            if len(local_operations) == 0:
-                print("n/a")
+        def print_dist_op(op: SqlOperation, is_single_node: bool) -> None:
+            if not is_single_node:
+                print(f"Distributed op: {op.format_sql()}")
+            else:
+                print("Skipped dist operation - single node cluster")
 
-            for op in local_operations:
-                print(op.format_sql())
+        for op in ops:
+            cluster = get_cluster(op._storage_set)
```

Comment:
why not move this line and the line below into the `print_dist_op` since it seems like you only need to get the cluster when we have a dist op?

Reply:
moved